### ISSUE TITLE
Installation of the second instance of bcbio 1.1.0 + mm10 genome: no whamg

### ISSUE THREAD

**Issue by naumenko-sa** [bcbio/bcbio-nextgen#2496]
Hello, bcbio developers and users!

Thanks for the great pipeline!

I'm installing the second instance of bcbio for my colleague on the same cluster.

Before installation:
- I modified PYTHONPATH and PATH, they were not pointing to the existing bcbio installation
- I removed ~/.conda/environments.txt to avoid mixing conda environments

Installation:
```
python bcbio_nextgen_install.py /hpf/largeprojects/lauryl/bcbio110 --tooldir=/hpf/largeprojects/lauryl/bcbio110 --genomes mm10 --aligners bwa --isolate
bcbio_nextgen.py upgrade -u skip --tools --tooldir /hpf/largeprojects/lauryl/bcbio110
```
Test
```
details:
- algorithm:
    aligner: bwa
    realign: false
    recalibrate: false
    remove_lcr: true
    save_diskspace: true
    svcaller: [lumpy, manta, cnvkit, metasv, wham]
    tools_on:
    variantcaller: false
  analysis: variant2
  description: '912R_A337376'
  files:
  - /hpf/largeprojects/lauryl/bcbio110/test/input/test_test_1.fq.gz
  - /hpf/largeprojects/lauryl/bcbio110/test/input/test_test_2.fq.gz
  genome_build: mm10
resources:
  default:
    cores: 2
    jvm_opts:
    - -Xms750m
    - -Xmx7000m
    memory: 7G
upload:
  dir: ../final
```
Error:
```
/bin/bash: whamg: command not found
' returned non-zero exit status 127
```

Indeed, there is no whamg in 
/hpf/largeprojects/lauryl/bcbio110/bin
/hpf/largeprojects/lauryl/bcbio110/anaconda/bin

However, wham is installed:
/hpf/largeprojects/lauryl/bcbio110/anaconda/pkgs/wham-1.8.0-hd28b015_0

There are wham and whamg binaries in the other bcbio installation:
/hpf/largeprojects/ccmbio/naumenko/tools/bcbio/bin/wham
/hpf/largeprojects/ccmbio/naumenko/tools/bcbio/bin/whamg

Could you please share any suggestion how to get
/hpf/largeprojects/lauryl/bcbio110/ installation running?

Thanks!
Sergey

**Comment by chapmanb** [comment]
Sergey;
Apologies about the issue. It looks like this was due to the wham bioconda package inadvertently being rolled back to an older version. The versioning on wham is confusing as there hasn't been a new release or updated git tag in a while, so you were getting the older version without whamg. I pushed a fix for this so an update should hopefully fix it:
```
bcbio_nextgen.py upgrade --tools
```
This doesn't have anything to do with your multiple installs as far as I can tell and is just a bug on the wham install, so hopefully this'll get your install working.
